src/main/java/net/mcreator/element/converter/fv28/FoodToItemConverter.java

```diff
@@ -58,7 +59,8 @@ public GeneratableElement convert(Workspace workspace, GeneratableElement input,
 			if (food.get("specialInfo") != null)
 				food.getAsJsonArray("specialInfo").iterator()
 						.forEachRemaining(element -> specialInfo.add(element.getAsString()));
-			item.specialInfo = specialInfo;
+			item.specialInformation.setFixedText(
+					specialInfo.stream().map(info -> info.replace(",", "\\,")).collect(Collectors.joining(",")));

```

Comment:
Food-to-item converter is already released, you'd better make a new one.

Reply:
What??? What if someone comes from a very old version of MCreator where the food mod element is still there?